### PR TITLE
Add napari_video recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Depends on:
- https://github.com/conda-forge/staged-recipes/pull/17762

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@postpop, do you agree with being added as a maintainer?

**Comment by postpop** [comment]
@goanpeca: Yes, happy to be added as maintainer.

**Comment by goanpeca** [comment]
Thanks a lot @postpop :)

File: recipes/napari_video/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: 333c3754be72847f6265e8e955b5cd296a8e521e46115e410c115ff39210fa92
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - flit
```
Comment (jaimergp) [inline recipes/napari_video/meta.yaml:19]:
```suggestion
    - flit-core
```

File: recipes/napari_video/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - flit-core
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - python >=3.6
+    - pyvideoreader
```
Comment (jaimergp) [inline recipes/napari_video/meta.yaml:25]:
```suggestion
    - pyvideoreader
    # adding dask-core due to unused import; remove in next release!
    - dask-core
```

File: recipes/napari_video/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - napari_video
```
Comment (jaimergp) [inline recipes/napari_video/meta.yaml:29]:
```suggestion
    - napari_video.napari_video
```

**Comment by jaimergp** [comment]
@postpop I think you have a [leftover dask.delayed import in your code](https://github.com/janclemenslab/napari-video/blob/main/src/napari_video/napari_video.py#L4) which will force us to add it to the requirements, but it should be removed in the next release.

**Pull request merged**